openvpn/utils.go

```diff
@@ -15,8 +15,8 @@ const finenameCharset = "1234567890abcdefghijklmnopqrstuvwxyz"
 func tempFilename(directory, filePrefix, fileExtension string) (filePath string) {
 	for i := 0; i < 10000; i++ {
 		filePath = directory + "/" + filePrefix + randomString(10, finenameCharset) + fileExtension
-		if _, err := os.Stat(filePath); os.IsExist(err) {
-			continue
+		if _, err := os.Stat(filePath); os.IsNotExist(err) {
```

Comment:
@Waldz currently, this loop is happening 10k times, no matter whether we find existing files or not (try to simulate the execution of lines).

This class does not contain any tests and it's coupled to OS, so I think it's ok for this fix to be merged without tests, because it's very critical!